packages/KCard/KCard.vue

```diff
@@ -1,35 +1,42 @@
 <template>
-  <div
+  <section
     :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏I think we can add [`aria-labeledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) here and have it point to the title attribute. This might require the unique id generator though.

Also, [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute) and point it to the body.

Reply:
The logic now uses `aria-label` if the `title` prop is specified. If not, it will generate a unique id for the title `div` and use that as the `aria-labelledby`. Generate a unique id for content and use as `aria-describedby`.